### PR TITLE
利用調査画面のデザインと送信内容の修正

### PR DESCRIPTION
**Pull request by cocoa-dev003**
## Issue 番号 / Issue ID

- Close #

## 目的 / Purpose

- 利用調査画面のデザイン修正
- デザインに合わせて送信する内容も修正

## 変更内容 / Changes

- 利用調査画面のデザイン修正
  - 回答の選択肢は修正していません 
- デザインに合わせて送信する内容も修正
  - json の値が null の時フィールドを含めるかどうかは検討中

## 破壊的変更をもたらしますか / Does this introduce a breaking change?

```
[] Yes
[x] No
```

## Pull Request の種類 / Pull Request type

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## 確認事項 / What to check

- [ ] 

## その他 / Other information

- 

---

Internal IDs:

- PBI 8856

### CODE REVIEW

File: Covid19Radar/Tests/Covid19Radar.UnitTests/Services/SurveyServiceTests.cs [hunk 1]
```diff
@@ -64,40 +66,40 @@ public async Task BuildSurveyContentTests_HasExposureDataProvision()
                         }
                     }
                 });
+            _mockUserDataRepository.Setup(x => x.GetStartDate()).Returns(new DateTime(2022, 1, 1, 0, 0, 0, DateTimeKind.Local));
```
Comment (ghost) [inline Covid19Radar/Tests/Covid19Radar.UnitTests/Services/SurveyServiceTests.cs]: GetStartDateはUTCで返されるはずです

**Comment by 01235110219565** [comment]
[Personal data protection and privacy policy    Last revision: May [31], 2021    Didi attaches great importance to the protection of your personal data. If your personal data is protected by the Personal Data (Privacy) Ordinance (Chapter 486 of the laws of Hong Kong), please refer to the following privacy policy (hereinafter referred to as "this privacy policy"). We will collect, hold, use, transfer and protect your personal data in accordance with this privacy policy. Before you use Didi's products and services or provide any personal data to Didi, please read carefully and fully understand and agree to this privacy policy. For terms of this privacy policy that have a significant relationship with your rights, we have bold the font to remind you of your attention.When you browse and access Didi platform and/or use any product or service in Didi platform (including Didi proprietary services and information services provided by Didi platform cooperative service providers), you agree that we collect, use, transfer and protect your personal data in accordance with this privacy policy.If you do not agree to the terms of this privacy policy, we may not be able to provide you with our services.  In this privacy policy, we refer to Didi (Hong Kong) Technology Co., Ltd. and its parent company and its affiliates; And you refer to Didi platform users. "Didi platform" refers to Didi taxi information platform (including but not limited to versions through PC version, mobile device application, application built-in Applet, etc.).  We collect, hold, use, transfer and protect your personal data in compliance with Hong Kong laws and your consent. Our purpose is to provide you with products and services in Didi platform and continuously improve the quality of such products and services, including but not limited to supporting us to carry out marketing activities related to products and services in Didi platform, improving the functions of existing products and services, develop new products or services, etc.  Scope of personal data collection  Personal data refers to any data that directly or indirectly identifies your personal identity or can directly or indirectly identify your personal identity. When you use Didi platform, you are not obliged to provide us with any information.  We will collect different types of personal data according to your requirements or the services and products we provide to you.You agree to provide or allow us to collect your following personal data:  -Identity information, including but not limited to your name, contact address, age and telephone number;  -If the user is a driver, in addition to identity information, we will also collect information about your company (if any), vehicles, vehicle registration records, driving licenses, vehicle insurance, etc;  -Your location and destination information, in order to fully realize our service functions, we will collect your background location information when we may need to participate, including to ensure your safety; You can disable the "location" permission in the device settings, thus refusing us to collect your location information;  -Record your common address so that you can enter the address quickly when you need to ride;  -Your payment information on Didi platform, such as credit card information and bank account information;  -Platform operation information, including but not limited to your IP address and browser details (which may involve the collection of cookies and network beacons. For details, see the use of cookies and network beacons section);  -Travel information, including but not limited to your departure place, destination, route, passing place and mileage information;  -Payment information, including but not limited to your payment time, amount, payment tools, bank account and payment account;  -Personal credit information related to platform operation, including but not limited to any credit status, credit score and other information that you pay through this platform;  -Call information and records, including but not limited to the number of call hours, dialed and answered mobile phone numbers and other information that you use Didi products and services;  -Other personal data collected according to the needs of our specific products and services, including but not limited to your opinions and suggestions on us and our products or services, mobile application software that you have used or frequently used, usage scenarios and usage habits, etc. (e.g. access to and read the permissions and reading of photos, films or other media, the permission to modify and delete the content of SD card is convenient for us to provide you with online customer service and complaint service, the permission to download files for updating version, read your contact information to facilitate us to contact your emergency contact person or call a car on behalf of others, find out the account on the device to pay for the car, the permission to use microphone to facilitate your use of voice call a car, and scan two-dimensional Code (QR Code) and control the permissions of The Flash). Depending on your mobile phone's privacy and permission settings for Didi platform, you agree that Didi can directly access such information from your mobile phone.  Certain personal data (such as your personal profile and contact information) are required for specific services. If you fail to provide such personal data for specific service requirements, we may not be able to provide you with relevant services or products. If you provide us with personal data, you agree and confirm that you have voluntarily provided us with relevant personal data in the context of fairness and reasonableness and reading this privacy policy.  Cookie and network Beacon Usage  Didi's platform uses a variety of technologies to collect other information, such as cookies, network beacons, and similar mechanisms, to help Didi understand the usage of the platform. Specifically, when you use the platform, Didi or a third-party service provider may automatically collect information about your technology and browsing through such technology, including: your computer or device type_operating system version_browser type and version, user agent field **Internet connection type** mobile network provider **date and time of your access** last time of your access **web page browsed by your browser** link clicked **search on the website** Internet Protocol (IP) address used to access Didi website **your general geographical location (such as city, state, postal code, or metropolitan area)** And the websites visited before using Didi website.  Accuracy of personal data  It is your responsibility to ensure the accuracy and completeness of all personal data you provide.  Methods for collecting personal data  When you agree to provide your personal data, you authorize us to collect your personal data through the following methods:  -We will collect and store personal data that you actively provide to us when you browse, access Didi platform and/or use Didi's products or services;  -We will collect and store personal data related to you recorded in the process of providing Didi's products or services to you;  -We will collect and store personal data that you actively submit or return to Didi through our customer service personnel and/or other channels;  -Before such third parties confirm that the personal data collected from you can be used for such purposes, we will collect and store legally obtained personal data related to you from affiliates, business partners and third parties independent data sources.  Purpose of personal data  You agree and authorize us to use your personal data for the following purposes:  -Provide you with Didi's products and services, and manage and optimize Didi's related websites and applications;  -Assist you in payment and payment on Didi's platform;  -Improve and improve the functions and quality of Didi's existing products and services, including but not limited to personalized settings and updates of products and service contents;  -Carry out market activities related to Didi products and services, including promoting the latest market activity information, preferential plans and brand-new products and services to you through SMS and in-terminal messages;  -Design and develop brand-new products and services and involve you in surveys and other activities on our products and services;  -Improve the security of Didi products and services, including but not limited to identity verification, customer service, security prevention, fraud monitoring, archiving and backup;  -Provide your personal data to assist relevant administrative, judicial or law enforcement agencies in conducting investigations under the requirements of relevant laws and other regulations;  -Maintain and develop our business systems and infrastructure, including testing and upgrading such systems, software verification or software upgrade;  -Monitor whether users use Didi's products and services and related information for fraudulent, illegal or illegal purposes;  -Other purposes notified to you at the time of collecting personal data and other purposes related to any of the above purposes.  In addition, we may send you information and notifications related to the above purposes, including but not limited to sending verification codes necessary to ensure service completion by SMS or in-terminal messages, push notifications necessary to use products or services, current car fee discounts and relief information, news about Didi products or services, market activities and promotional information.  The retention time of your personal data will not be longer than the actual time required to meet the above purposes.  Disclosure and sharing of personal data  We have the obligation to keep your personal data confidential. Without your consent, we will not provide your personal data to any third party for any purpose. If it is necessary to disclose the information for the purpose of collecting or directly related purposes, you understand, agree and authorize us to transfer or share your personal data to or with the following third parties through the use of Didi's services or products and the provision of relevant personal data to US:  -In order to improve the quality of our products and services or provide you with brand-new products and services, we will transfer or share your relevant personal data among our affiliates, or provide your reasonable and necessary personal data to a third party for analysis and statistics to a necessary extent; Personal data and big data used for analysis and statistics will be anonymous and integrated. Such data alone cannot be traced back to individual individuals;  -If some products and services you use through Didi platform are provided by our partners or jointly provided by us and our partners or suppliers (including but not limited to any third-party payment service provider or other service provider), we will share or transfer the personal data necessary to provide you with the corresponding products and services to them; including providing your basic personal information, credit card or bank account information necessary to the payment agent or the payment platform cooperating with Didi to assist you in making payment on Didi platform;  -In order to carry out joint promotion activities with third parties, we may share or transfer personal data necessary for carrying out activities with them;  -In order to protect your legitimate rights and interests, when assisting in handling transaction disputes or disputes related to you, we will provide your party or a third party with the personal data necessary for resolving transaction disputes or disputes;  -Any proposed or actual participant, assignor or assignee of all or part of our operations or business, including but not limited to Didi platform;  -Organizations that manage our business and corporate strategies (and for such purposes), including organizations that participate in the transfer or sale of all or part of our assets or businesses, and organizations that participate in the management of our corporate risks and financing functions; And  -According to relevant laws and regulations, law enforcement agencies shall set procedures to collect necessary personal data in order to investigate crimes; Or administrative agencies, judicial agencies or law enforcement agencies shall set procedures according to law and have the right to collect necessary personal data in accordance with relevant laws. We will provide them with your relevant personal data.  We will not provide your personal data to any third party unless you agree with the above.  We may also occasionally use unidentified comprehensive information about customers to further improve the design of the services we provide and the services themselves. Such information does not identify the identity of any specific individual and does not constitute personal data in relevant laws.  Based on the purpose or transfer of the above-mentioned personal data and the transfer of some data may involve cloud technology, you agree that your relevant personal data may be transferred and stored in a local or national server outside the Hong Kong Special Administrative Region. When considering whether to transfer your personal data to such countries, Didi will consider whether such places or countries provide the same or higher personal data protection standards as Hong Kong laws.  Data security and protection measures  -We will adopt strict security systems and prevailing security technologies and procedures in the industry to ensure that your personal data will not be lost, leaked, damaged, abused or altered (unless you agree or require);  -We will use security technologies and procedures prevailing in the industry to protect your personal data from unauthorized access and use;  -Our employees and service contractors (including the relevant third parties mentioned in the "disclosure and sharing of personal data" section) will be bound by the confidentiality agreement, and will also be subject to Didi's permission control and operation monitoring of your personal data in accordance with this privacy policy.  However, please note that any security system has possible and unknown risks.  Your party, the third-party website operator you use, the third-party service provider you use, and the third party that obtains your personal data through US may have their own privacy protection policies and methods and measures to obtain your personal data. Such third-party privacy protection policies, methods and measures to obtain personal data will not be controlled and monitored by us. Although we will sign confidentiality agreements with third parties such as partners and suppliers who may come into contact with your personal data, and make reasonable efforts to urge them to fulfill their confidentiality obligations, we cannot guarantee that third parties will take confidentiality measures according to our requirements, and we will not bear any legal liability for the behavior, use and consequences of personal data of third parties. Therefore, when you use the services of third-party service providers or provide personal data to third-party service providers, you should always be vigilant and carefully read the privacy policies and personal data collection statements of such third parties.  Without the consent of parents or guardians, we will not knowingly collect or store personal data of people under the age of 18 in the database.  As a user of products and services of Didi platform, you can decide whether to use products or services of Didi platform and whether to provide personal data on your own initiative.  Personal data used for direct promotion purposes  If you agree, we can also use your personal data to send you direct promotional materials (using the information in our services, via email, telephone, SMS, email or other means) to provide or publicize our products and services and/or select third-party products and services. Such products and services (of us or third parties) include:  -Products and services related to the sales of the following items by us and our affiliates, subsidiaries and business partners:(1) car booking service;(2) banking, financial or payment services;(3) transportation and tourism;(4) loyalty and reward programs;(5) social media, entertainment, e-commerce, information and communication software and services; And  -The above-mentioned services and products as well as the third-party suppliers related to the following services:(1) Online car booking services;(2) banking, financial or payment services;(3) transportation and tourism;(4) loyalty and reward programs;(5) social media, entertainment, e-commerce, information and communication software and services.  If you do not want us to use your personal data for the above promotion purposes, we will respect your requirements. You can make the request by: (I) notifying us (you can send an email to: hkcs@didiglobal.com); Or (ii) making the request according to the relevant instructions in our promotional communication, including clicking on the message we send to you or the unsubscribed link contained in the email.  Please note that we will not share your personal data with any advertisers unless your consent has been obtained.  For the avoidance of doubt, the above agreement is separate and independent of the service you registered with us. Therefore, consent can be revoked at any time in accordance with this privacy policy, without considering the duration of the service, and any applicable service contract still exists after termination or expiration.  View, correct, and delete personal data  You have the right to consult, correct or delete the personal data we hold about you. If you want to request any copy of the personal data you have provided, or if you think that the personal data we have collected and saved about you is inaccurate and wish to correct it, please write to the following address, but deleting some of the personal data we have provided may cause us to be unable to provide some or all of the services.  If you want to consult or correct your personal data, or ask for information about our personal data policies and practices, or the types of personal data we hold, please email to the following address: hkcs@didiglobal.com.  We will respond to all requests made to us pursuant to the reference, correction and deletion of personal data in this section within the time required by applicable laws and regulations. You must provide satisfactory proof and/or explanation of the inaccuracies in the information, and we will consider revising the specific information.  According to the provisions of the Personal Data (Privacy) Ordinance (Chapter 486 of the laws of Hong Kong), we have the right to charge a reasonable fee for data access requirements.  Changes and updates to this privacy policy  With the further improvement of information services on the network platform, the content of this privacy policy will be updated from time to time. Once the updated privacy policy is published on this application or Didi website, it will be effective and replace the original privacy policy. We encourage you to check this page regularly to learn about our latest policies on privacy protection.  If we think any changes to this privacy policy are significant, we will notify you before the changes take effect (through the network platform, directly contact you or otherwise). If you continue to use our services after any changes to this privacy policy, whether or not you receive our notice, it means that you agree to the revised privacy policy.]() @halsk

File: Covid19Radar/Tests/Covid19Radar.UnitTests/Services/SurveyServiceTests.cs (hunk 1 above)
Comment (cocoa-dev003) [reply Covid19Radar/Tests/Covid19Radar.UnitTests/Services/SurveyServiceTests.cs]: ありがとうございます、修正しました。

Comment (ghost) [reply Covid19Radar/Tests/Covid19Radar.UnitTests/Services/SurveyServiceTests.cs]: 以下の意図として、UTCの2022/01/01 00:00で良いですか？
日本時間の2022/01/01 00:00とするなら、-9時間したほうがいいと思います。
> _mockUserDataRepository.Setup(x => x.GetStartDate()).Returns(new DateTime(2022, 1, 1, 0, 0, 0, DateTimeKind.Utc));

**Review by cocoa-dev009 — APPROVED** [review]

File: Covid19Radar/Tests/Covid19Radar.UnitTests/Services/SurveyServiceTests.cs (hunk 1 above)
Comment (cocoa-dev003) [reply Covid19Radar/Tests/Covid19Radar.UnitTests/Services/SurveyServiceTests.cs]: はい、日本時間は特に考えずUTCの2022/01/01 00:00を設定しています。

**Review by ghost — APPROVED** [review]

**Pull request merged**